src/test/kotlin/com/hedvig/accountservice/transaction/FailIfNoTransaction.kt

```diff
@@ -0,0 +1,28 @@
+package com.hedvig.accountservice.transaction
+
+import com.hedvig.accountservice.schedulers.database.SubscriptionScheduledRepository
+import org.jdbi.v3.core.Jdbi
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.junit4.SpringRunner
+import java.time.Month
+import java.time.Year
+
+
+@RunWith(SpringRunner::class)
+@SpringBootTest
+class FailIfNoTransaction {
+
+
+    @Autowired
+    lateinit var jdbi: Jdbi
+
+    @Test
+    fun test() {
+        jdbi.withHandle<Boolean, RuntimeException> { h ->
+            h.attach(SubscriptionScheduledRepository::class.java).checkExists("123", Year.of(1991), Month.JANUARY)
+        }
+    }
```

Comment:
What is the assertion here? Shouldn't it crash?
```suggestion
    @Test(expected = MegaCrashException::class)
    fun test() {
        jdbi.withHandle<Boolean, RuntimeException> { h ->
            h.attach(SubscriptionScheduledRepository::class.java).checkExists("123", Year.of(1991), Month.JANUARY)
        }
    }
```

Reply:
sry a test that should not be there!